### DISCUSSION TITLE
Split & Panel - Lazy Loading

### DISCUSSION THREAD

**Discussion by photojo** [filamentphp/filament#5607] [Help] [answered]
I use filamentphp table with split and panel functionality with "->collapsible()".
In panel I use view::make to implement a view. The view is very huge. So it takes several seconds until the hole page is displayed.

`  
Panel::make([
               View::make('cars')->view('filament.tables.columns.subTable'),
                ])
                ->collapsible()`
How can I downsize the loading time? 
Is is possible to render view just when I expand the Panel?

**Comment by danharrin** [comment, marked as answer]
Maybe use an action modal instead of a collapsible panel? That only gets loaded when open

**Reply by photojo** [reply]
Ah, ok.
Thank you very much.
